packages/client/src/store/actions/bookingOperations.ts

```diff
@@ -222,27 +223,28 @@ interface sendICSFile {
 }
 
 export const sendICSFile: sendICSFile =
-  ({ icsFile, email, secretKey, displayName, name }) =>
+  ({ icsFile, email, secretKey, displayName = "displayName", name = "name" }) =>
```

Comment:
`displayName` should most definitely not fall back to `"displayName"` (should probably be organization name)

Reply:
Same goes for `name`